### ISSUE TITLE
Multidimensional Anomaly Detection

### ISSUE THREAD

**Issue by seanlaw** [TDAmeritrade/stumpy#1030]
There is a new paper [here](https://arxiv.org/pdf/2409.09298)

**Comment by seanlaw** [comment]
> I'm not sure if you still remember me

Yes, of course I remember you @SaVoAMP! I hope all is well and congratulations on your Masters thesis.

> Have you already had a chance to take a closer look at the paper or perhaps implemented anything from it?

No, we haven't had a chance to study this yet as other tasks have taken a higher priority. Having said that, we are certainly interested in at least confirming, as a first step, that the paper is reproducible. Will you consider creating a notebook that could possibly reproduce the work? 

> I always found the matrix profile papers to be of much higher quality than most of the work I've encountered in the machine learning field so far.

I completely agree!

**Comment by seanlaw** [comment]
That sounds like a good plan! Please don't feel pressured to take this on if you already have a lot on your plate. There is absolutely no rush to get started but I'd like to resolve it promptly once we choose to commence (and not let it drag out in time). Thanks again for thinking of us!

**Comment by SaVoAMP** [comment]
That sounds very reasonable. I'll take a look at the paper when I find a bit of time and will let you know once I have a better idea of whether I can contribute.
